lib/appium_lib/device/device.rb

```diff
@@ -15,6 +15,7 @@ module Device
         device_locked?:       'session/:session_id/appium/device/is_locked'
       },
       get:  {
+        get_device_time:        'session/:sessionId/appium/device/system_time',
```

Comment:
We should follow Ruby coding conventions and not use `get` prefixes in method names. So the better name would be `device_time`.